config/application.yml.example

```diff
@@ -62,6 +62,8 @@ development:
   equifax_avs_username: 'sekret'
   equifax_eid_username: 'sekret'
   equifax_endpoint: 'sekret'
+  equifax_gpg_email: 'logs@login.gov'
```

Comment:
is this a real email address? if not, should we ref something obviously fake like `logs@example.com` ?